modules/how-etcd-works.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+// * etcd/etcd-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="how-etcd-works_{context}"]
+= How etcd works
+
+To ensure a reliable approach to cluster configuration and management, etcd uses the etcd Operator. The Operator simplifies the use of etcd on a Kubernetes container platform such as {product-title}. With the etcd Operator, you can complete the following tasks: 
+
+* Create or delete etcd members 
+* Resize clusters 
+* Perform backups
+* Upgrade etcd 
+
+Additionally, use the etcd Operator to deploy and manage etcd clusters for your applications. The etcd Operator:
```

Comment:
```suggestion
Additionally, use the etcd Operator to deploy and manage the etcd cluster for the openshift control-plane. The etcd Operator:
```
The cluster-etcd-operator only handles a singleton instance of the etcd cluster. And that is only used by the control plane, not the applications running on the cluster.